frontend/views/components/Page.vue

```diff
@@ -186,12 +186,12 @@ $pagePaddingDesktop: 5.5rem;
   & > span {
     max-width: 100%;
 
-    & > span {
+    &:not(:has(*)) {
       display: block;
       max-width: 100%;
     }
 
-    & > span,
+    &:not(:has(*)),
```

Comment:
What does `:has(*)` mean? I've never seen this before. /cc @SebinSong your thoughts also appreciated